R/list-combine.R

```diff
@@ -0,0 +1,59 @@
+#' Combine list elements into a single data structure
+#'
+#' @description
+#' * `list_c()` combines elements into a vector by concatenating them together
+#'   with [vctrs::vec_c()].
+#'
+#' * `list_rbind()` combines elements into a data frame by row-binding them
+#'   together with [vctrs::vec_rbind()].
+#'
+#' * `list_cbind()` combines elements into a data frame by column-binding them
+#'   together with [vctrs::vec_cbind()].
+#'
+#' @param x A list.
+#' @param ptype An optional prototype to ensure that the output type is always
+#'   the same.
+#' @param id By default, `names(x)` are lost. Alternatively, supply a string
+#'   and the names will be saved into a column with name `id`. If `id`
```

Comment:
This makes it sound like the column name is always named `id`. Perhaps:

> Alternatively, supply `id` with a column name into which the names will be saved.

Reply:
I'll use `{id}` since that's a convention we use elsewhere and should be familiar to many readers.